package.json

```diff
@@ -2,13 +2,14 @@
   "name": "promise-iterate",
   "version": "0.2.4",
   "description": "Iteration over cursor-like objects with async/await",
-  "main": "index.js",
   "scripts": {
-    "build": "rm -rf dist/node && babel src -d dist/node",
+    "build": "rm -rf dist/ && tsc",
     "ci": "npm run lint && npm test",
-    "lint": "eslint .",
-    "prepublishOnly": "npm run build",
-    "test": "jest"
+    "ci:commitlint": "commitlint-jenkins --pr-only",
+    "lint": "eslint . && tsc --noEmit",
+    "prepublishOnly": "npm run build && if [ \"$CI\" = \"\" ] && [ \"$npm_config_dry_run\" != true ]; then node -p \"JSON.parse(process.env.npm_package_config_manualPublishMessage)\"; exit 1; fi",
+    "test": "jest",
+    "watch": "rollup -cw"
```

Comment:
Same as in https://github.com/mixmaxhq/url-obfuscation/pull/293
Probably `rollup` is not needed here?

Reply:
it is not. I had it replaced before, but had to re-run the `mixmax dev repo typescript` command and I forgot to re-remove after it re-added rollup. amended on my last commit, tks!